src/utils/urls.js

```diff
@@ -127,20 +127,7 @@ export function createFullyQualifiedUrl(pathname) {
 }
 
 export function getSearchUrl(query) {
-  const url = new URL(document.location);
-  const keys = [];
-  for (const key of url.searchParams.keys()) {
```

Comment:
We need this for the case when users copy/paste a query. This was an original requirement.

Reply:
Never mind I think this was a problem when we had the exact tags, not the full text.

Reply:
Ah, I didn't refresh :-)

Reply:
yeah, I tried to reproduce the problem but I really think I set this up because of the tag search and the it got left over. Anyway on the search page if you come with a query string query=yourQuery&extraParam=something it returns a 400, I'm not sure what was the case back then where that could have happened when redirecting.